src/nvim/api/vim.c

```diff
@@ -1437,26 +1437,28 @@ void nvim_chan_send(Integer chan, String data, Error *err)
 ///                    end-of-buffer region is hidden by setting `eob` flag of
 ///                    'fillchars' to a space char, and clearing the
 ///                    |EndOfBuffer| region in 'winhighlight'.
-///   - `border`: style of (optional) window border. This can either be a string
-///      or an array. the string values are:
-///     - "none" No border. This is the default
-///     - "single" a single line box
-///     - "double" a double line box
-///     - "shadow" a drop shadow effect by blending with the background.
-///     If it is an array it should be an array of eight items or any divisor of
+///   - `border`: Style of (optional) window border. This can either be a string
+///      or an array. The string values are
+///     - "none": No border (default).
+///     - "single": A single line box.
+///     - "double": A double line box.
+///     - "rounded": Like "single", but with rounded corners ("╭" etc.).
+///     - "solid": Adds padding by a single whitespace cell.
+///     - "shadow": A drop shadow effect by blending with the background.  
```

Comment:
Lint failure is related to this line, where I added two spaces to introduce a line break (which is standard Markdown). Otherwise the following line -- which starts a new topic, array arguments -- runs into the final item of the list of valid string arguments.

If there is a better way of separating these two, I'm all ears.

Reply:
Couldn't find a way, so I just made the array arguments another bullet point.

(happy lint == happy life)

Reply:
Alternatively, you could have ignored this lint issue and then it would have become a known issue

Reply:
Ignoring lint sets a bad precedent ;)

(It didn't even work, so I saw no point introducing a lint failure that would pollute future PRs.)